src/mod_tls.c

```diff
@@ -233,6 +233,11 @@ static int wolfTlsInitSslDefaults(WOLFSSL_CTX* ctx, WOLFSSL** ssl)
         wolfSSL_SetIOReadCtx(tssl, cbCtx);
         wolfSSL_SetIOWriteCtx(tssl, cbCtx);
 
+    #if 0
```

Comment:
Maybe add a comment saying that if you enable this code, ensure HAVE_CURVE_X448 is defined or add guard inside the `#if 0`